### ISSUE TITLE
Actions must be plain objects. Use custom middleware for async actions.

### ISSUE THREAD

**Issue by tatyshev** [zalmoxisus/redux-devtools-extension#98]
After update on v1.1.0. Upon attempt to toggle some action, I get the following error:

`
Uncaught Error: Actions must be plain objects. Use custom middleware for async actions.
`

This error is reproduced only when redux devtools opened inside chrome devtools. In other cases, it works as before.

**Comment by zalmoxisus** [comment]
Sorry, forgot about chrome devtools in https://github.com/zalmoxisus/redux-devtools-extension/commit/1dc59e97a5362e900155c52902fe3f63d1a36294. Will fix ASAP.

**Comment by zalmoxisus** [comment]
Fixed in `v1.1.1`, should appear on Chrome store in about an hour. Thanks for the report!

**Comment by tatyshev** [comment]
Good job. Thank you!

**Comment by ValeryIvanov** [comment]
Hi! I still cannot use this feature, nothing happens, no errors in console...

Version says 1.1.1

**Comment by zalmoxisus** [comment]
@ValeryIvanov, do you get the same error? Could you please provide more details to reproduce it?

**Comment by ValeryIvanov** [comment]
No error. I see all actions pop up, but if I try to 'remove' some action, nothing happens. Commit, revert, sweep buttons does nothing also. So somehow this new state is not sent to my app?

Using Chrome 50.0.2661.87 m

**Comment by zalmoxisus** [comment]
Does this happen also from extension windows or only from inside chrome devtools? Did it work in `1.0`?

**Comment by ValeryIvanov** [comment]
This happens from both. I can actually make it do something but it is kinda messed up. For example I have multiple radio buttons. I use React btw and react-redux. So first radio is selected. If I click on the second one and then the third one then two actions are emmited whitch is all fine. Now if I cross out last (second) action, nothing happens. If I cross out first action, nothing happens as well. But if I enable second action right away, then second radio button becomes active, so only then this second action becomes valid. This strange behavior is happening on my teammates machine as well, same app, so this is not only my problem.

Two options. Either out project is set up badly (maybe, since we just started experimenting with Redux) or this plugin is not working correctly.

**Comment by ValeryIvanov** [comment]
Ok scratch that, I found out what was wrong at least I guess what was wrong. I was using custom made state object and I was trying to make a deep copy like that 

`const stateProto = Object.getPrototypeOf(state);`
`const newState = Object.assign(Object.create(stateProto), state);`

This custom object has actually getJsonObj function that creates simple object without any functions, only properties with their values and a constructor that uses the same getJsonObj to recreate this custom so called smart object. So I used that to create next new state and everything started working like it should.

Here is that example of that smart object


    class ExampleData {
        id : String;
    }
    
    export class Example {
        constructor(data : ExampleData) {
            if (data) {
                this._id = data.id;
            }
        }
        get id() : String {
            return this._id;
        }
        getJsonObj() : ExampleData {
            const exampleData = new ExampleData();
            exampleData.id = this.id;
            return exampleData;
        }
    }


and here is how I create new state object

`const newState = new Example(prevState.getJsonObj());`
